eventhubs/spring-cloud-azure-starter/spring-cloud-azure-sample-eventhubs-kafka/src/main/resources/application-rm.yaml

```diff
@@ -16,3 +16,9 @@ spring:
           group: $Default
         supply-out-0:
           destination: ${EVENTHUBS_KAFKA}
+      binders:
+        kafka:
+          environment:
+            spring:
+              main:
+                sources: com.azure.spring.cloud.autoconfigure.eventhubs.kafka.AzureEventHubsKafkaAutoConfiguration
```

Comment:
I remember it's not necessary for version since 4.4.1.
@yiliuTo Please confirm.

Reply:
this is required, the one that is not required is `com.azure.spring.cloud.autoconfigure.kafka.AzureKafkaSpringCloudStreamConfiguration`